docs/tech-reference/export-distributor.rst

```diff
@@ -62,6 +62,9 @@ Optional Configuration Parameters
  ISO images and published over HTTP or HTTPS. Instead, they are written to the export directory.
  This option is useful if exporting to an external hard drive, for example.
 
+``relative_url``
```

Comment:
actually, it is not working at all, the 'raletive_url' option is missing in the config of export distributor
```
$ curl -H "Accept: application/json" -X GET -k -u admin:admin 'https://localhost/pulp/api/v2/repositories/111/distributors/export_distributor/'|python -m json.tool
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   357  100   316  100    41   1050    136 --:--:-- --:--:-- --:--:--  1049
{
    "_id": {
        "$oid": "568bca9b45ef4863e7d7b2c3"
    },
    "_ns": "repo_distributors",
    "auto_publish": false,
    "config": {
        "http": false,
        "https": true
    },
    "distributor_type_id": "export_distributor",
    "id": "export_distributor",
    "last_publish": "2016-01-05T13:53:09Z",
    "repo_id": "111",
    "scheduled_publishes": [],
    "scratchpad": null
}